end-to-end-tests/specs/core/groupComparison.screenshot.spec.js

```diff
@@ -7,6 +7,7 @@ var checkElementWithTemporaryClass = require('./../specUtils').checkElementWithT
 var checkElementWithMouseDisabled = require('./../specUtils').checkElementWithMouseDisabled;
 
 const CBIOPORTAL_URL = process.env.CBIOPORTAL_URL.replace(/\/$/, "");
+const OverlapStrategy = require("./../../../src/pages/groupComparison/GroupComparisonStore").OverlapStrategy;
```

Comment:
@adamabeshouse this actually brok the whole spec file.   We can't import ts files into js because none of this is compiled for typescript.